.github/workflows/publish-release.yml

```
@@ -39,12 +39,33 @@ jobs:
         run: |
           yarn install
           yarn build
+          yarn build:docs
       - uses: actions/cache@v3
         id: restore-build
         with:
-          path: ./dist
+          path: |
+            ./dist
+            ./docs
           key: ${{ github.sha }}
 
+  publish-staging-to-gh-pages:
```

It's worth commenting here on why this step is being duplicated between here and the normal staging workflow

e.g.

```suggestion
  # Normally this happens during the `publish-main-docs` workflow.
  # That workflow will not run in this case because GitHub Actions workflows cannot be
  # triggered by events that were themselves triggered by a GitHub Actions workflow.
  publish-staging-to-gh-pages:
```

---

Actually... wait.... this isn't true. The staging publishing worked just fine for `abi-utils`, because the RC gets merged by a human.

We only need the `latest` and tag docs publishing step here. It's just the tag trigger that was broken, because the tagging is solely done by GitHub actions.

We can remove this then.